src/components/structures/auth/Registration.js

```diff
@@ -203,6 +204,20 @@ module.exports = React.createClass({
         });
     },
 
+    _requestEmailToken: function(emailAddress, clientSecret, sendAttempt, sessionId) {
```

Comment:
As mentioned in the js-sdk PR, just an idea, but would it make sense to pass `attemptAuth` the registration url too and move this into the `attemptAuth` method as well?

Reply:
hopefully clarified in the js-sdk PR (the interactive auth helper doesn't know what endpoint you're calling and so what requestToken endpoint to use).